guides/code-style/.rubocop.yml

```diff
@@ -19,24 +25,14 @@ Style/BlockComments:
 Style/Documentation:
   Enabled: false
 
-Style/EmptyLinesAroundBlockBody:
-  Exclude:
-    - 'db/schema.rb'
-
-Style/ExtraSpacing:
-  Exclude:
-    - 'bin/*'
-    - 'db/schema.rb'
-
 Style/FirstParameterIndentation:
   EnforcedStyle: consistent
 
 Style/GuardClause:
-  Enabled: false
+  Enabled: true
```

Comment:
We use guard clauses extensively. Unless I'm mistaken, this would prevent us from using them. can you confirm?

Reply:
Makes sense